nuclio_sdk/test/test_event.py

```diff
@@ -35,7 +35,7 @@ def test_event_to_json_bytes_body(self):
             serialized_event.trigger.__dict__,
             {"kind": "http", "name": "my-http-trigger"},
         )
-        self.assertEqual(serialized_event.last_in_batch, False)
+        self.assertFalse(serialized_event.last_in_batch, False)
```

Comment:
The second arg is intended to be a message

see https://www.geeksforgeeks.org/python-unittest-assertfalse-function/